internal/controllers/usages.go

```diff
@@ -20,6 +20,7 @@ type Usage struct {
 	ResourceName string  `json:"resource_name"`
 	UsageValue   float64 `json:"usage_value"`
 	UpdateType   string  `json:"update_type"`
+	Metadata     string  `json:"metadata"`
```

Comment:
Minor question for someone who's worked more with this code is if this should be any other type or what. I guess this should mean that we insert blank strings rather than nulls, which is probably fine but maybe someone has a different thought there. Haven't worked to integrate with anything yet either, of course.